servant-examples/digest-auth/digest-auth.hs

```diff
@@ -0,0 +1,33 @@
+{-#LANGUAGE DataKinds, TypeOperators, OverloadedStrings #-}
+module Main where
```

Comment:
I accidentally added this file in one of the commits.   I'll remove it again as it's not suppoted to be in this pull.